src/pages/collections/[collectionAddress]/activity.tsx

```diff
@@ -0,0 +1,319 @@
+import {
+  CollectionPageProps,
+  getCollectionPageServerSideProps,
+} from '@/views/collections/collections.utils';
+
+import CollectionLayout from 'src/views/collections/CollectionLayout';
+import { InView } from 'react-intersection-observer';
+
+import { GetServerSideProps } from 'next';
+
+import { uniq } from 'ramda';
+import React, { FC, ReactNode, useState } from 'react';
+import { NftActivity, useCollectionActivitiesQuery } from 'src/graphql/indexerTypes';
+import TopLevelFilterButton from 'src/components/TopLevelFilterButton';
+import clsx from 'clsx';
+import NoProfileItems, { NoProfileVariant } from '../../../components/NoProfileItems';
+import { ApolloQueryResult, OperationVariables } from '@apollo/client';
+import { None } from '../../../components/OfferForm';
+import { TailSpin } from 'react-loader-spinner';
+import { format } from 'timeago.js';
+import Link from 'next/link';
+import { DollarSign, Tag } from 'react-feather';
```

Comment:
I think we are moving towards using heroicon for iconography for now

Reply:
I updated Tag icon to use heroicons library, but for dollar icon the icon we need is not present in heroicons.

Reply:
gotcha